docs/overview.md

```diff
@@ -61,156 +58,141 @@ Note: The Processing model will likely evolve over time alongside concurrency, t
+Tremor pipeline operators will assume _some_ structure. For hierarchic or nested formats such as JSON and MsgPack, Tremor uses `serde` serialisation and deserialisation capabilities.
 
-Tremor pipeline operators however, often assume _some_ structure. For hierarchic or nested formats such as JSON and MsgPack, Tremor uses the `serde` serialisation and deserialisation capabilities.
+Therefore, the _in-memory_ format for _JSON-like_ data in Tremor is effectively a `simd_json::Value`. This provides the advantage of allowing Tremor-script to work with YAML, JSON or MsgPack data with no changes or considerations in the Tremor-script based on the origin data's format.
 
-Therefore, the _in-memory_ format for _JSON-like_ data in Tremor is effectively a `simd_json::Value`. This has the advantage of allowing Tremor-script to work against YAML, JSON or MsgPack data with no changes or considerations in the Tremor-script based on the origin data format.
+For line-oriented formats such as the Influx Line Protocol or GELF, they are typically transformed into Tremor's in-memory format (currently based on `serde`).
 
-For line oriented formats such as the Influx Line Protocol, or GELF these are typically transformed to Tremor's in-memory format ( currently based on `serde`).
+For raw binary or other data formats, Tremor provides a growing set of codecs that convert external data into Tremor in-memory form or vice-versa.
 
-For raw binary or other data formats, Tremor provides a growing set of codecs that convert external data to Tremor in-memory form or that convert Tremor in-memory form to an external data format.
+In general, operators and developers should _minimize_ the number of encoding/decoding steps required for the transit of data through Tremor or between Tremor instances.
 
-In general, operators and developers should _minimize_ the number of encoding and decoding steps required in the transit of data through Tremor or between Tremor instances.
-
-The major overhead in most Tremor systems is encoding and decoding overhead. To compensate that, as JSON is the most dominant format, we [ported](https://github.com/simd-lite/simdjson-rs) [simd-json](https://github.com/lemire/simdjson) this reduces the cost of en- and decoding significantly compared to other JSON implementations in Rust.
+Since the major overhead in most Tremor systems is encoding and decoding and JSON is the dominant data format, we [ported](https://github.com/simd-lite/simdjson-rs) [simd-json](https://github.com/lemire/simdjson) to reduce the cost of encoding and decoding significantly compared to other JSON implementations in Rust.
 
 ### Distribution model
 
-Tremor does not ( yet ) have an out-of-the-box network protocol. A native Tremor protocol is planned in the immediate / medium term.
+Tremor does not yet have an out-of-the-box network protocol. A native Tremor protocol is planned for development in the immediate/mid-term.
```

Comment:
I think medium-term is more appropriate as it references a  period of time in the near future.